src/controller/config.js

```diff
@@ -208,31 +175,65 @@ const getClickCoordinates = (event) => {
 
     const x = event.clientX - rect.left;
     const y = event.clientY - rect.top;
+    const ctx = imgCanvas.getContext("2d");
 
     currCoordinates = {
         x: x,
         y: y
     }
+    setupAddBodyTag()
+    for(let key in coordinatesMap){
```

Comment:
check if we right clicked on an existing body tag. If so then prompt for editing